content/en/releases/platforms.md

```diff
@@ -0,0 +1,307 @@
+(`GOOS`) as "platforms". Target of this document is to outline different
+categories of platforms as well as guiding through their graduation criteria.
+
+How to introduce new supported architectures and operating systems is outline in
+the the [platforms guide](#platforms-guide).
+
+<!-- body -->
+
+## Tiers
+
+Build and release support for different platforms' artifacts are organized into
+three tiers, whereas each comes with a different set of guarantees. Tiers can be
+scoped to single binaries or a subset of them. This means for example that the
+project can provide Tier 1 support for client binaries, even if the server
+binaries do not exist at all.
+
+### Tier 1
+
+Tier 1 platforms are considered as "expected to work". To achieve this, they
+have to fulfill the following criteria:
+
+- Official binary releases are provided for the platform. This includes
+  binaries / executables, container images, as well as deb and rpm packages.
+  Building the artifacts is integrated in the release process.
+- Continuous Integration is set up to run tests for the platform. Necessary
+  tests are defined by SIG Release and usually correspond to the
+  [`blocking`](https://testgrid.k8s.io/sig-release-master-blocking) and
+  [`informing`](https://testgrid.k8s.io/sig-release-master-informing) testgrid
+  dashboards. For Tier 1 only the `blocking` jobs are required, the `informing`
+  ones are optional.
+- Documentation about the usage of artifacts for the platform is available.
+
+### Tier 2
+
+Tier 2 platforms are considered as "expected to build". To achieve this, they
+have to fulfill the following criteria:
+
+- Official binary releases are provided for the platform. Building the artifacts
+  is integrated in the release process.
+- Automated testing is not or only partially set up.
+
+It may be possible that single features are not available for a certain
+platform.
+
+### Tier 3
+
+Tier 3 platforms are those that have been demonstrated to work and have a
+documented build process that is available to anyone in the community. There is
+no guarantee that builds will continue to work, and the platform may be dropped
+if documentation is not maintained.
+
+- Official builds are not available.
+- Automated testing is not setup.
+- Documentation on how to build for the platform is available.
+
+## Currently available Kubernetes platforms
+
+The following table defines the current setup of available Kubernetes platforms:
+
+| Platform        | Tier 1 | Tier 2 | Tier 3 |
+| --------------- | :----: | :----: | :----: |
+| `amd64-linux`   |   ✅   |        |        |
+| `arm64-linux`   |        |   ✅   |        |
+| `amd64-darwin`  |        |   ✅   |        |
+| `386-linux`     |        |   ✅   |        |
+| `ppc64le-linux` |        |   ✅   |        |
+| `s390x-linux`   |        |   ✅   |        |
+| `386-windows`   |        |   ✅   |        |
+| `amd64-windows` |        |   ✅   |        |
```

Comment:
Is there a reason why windows isn't `Tier 1`?

I believe all of the requirements are met.

* for build artifacts, all of the components needed to join windows nodes to the cluster (kubelet, kubeadm, kubectl) are produced as part of official releases.
* We have CI test up that is monitored by the release team (https://testgrid.k8s.io/sig-release-1.31-informing#capz-windows-1.31)
* We have a lot of documentation on k8s.io w.r.t. managing Windows nodes in a cluster

Reply:
Hm, right now we state:

> For Tier 1 only the `blocking` jobs are required, the `informing` ones are optional

Which technically would mean that we need to have all the jobs available in blocking for linux/amd64 also for windows. Would that make sense? Maybe we should think about creating a tier-based dashboard view. (cc @kubernetes/release-managers)

Reply:
> all of the components needed to join windows nodes to the cluster (kubelet, kubeadm, kubectl) are produced as part of official releases.

This is not completely true. We still don't produce binaries kube-apiserver, kube-controller-manager, kube-scheduler capable to run to Windows environments.

Reply:
So is it a requirement that we be able to build / run / test the control plane components for a given platform for it to be `Tier 1` supported?

Also, those components (kube-apiserver, kube-controller-manager, kube-schedule) aren't required to join a node to an existing cluster. We do build / test all of the the componenets required to join a node to the cluster, just not to have to run control-plane components on those nodes.